.github/ISSUE_TEMPLATE/incident.yml

```diff
@@ -0,0 +1,30 @@
+name: Incident 🚒
+description: Respond to a production incident - checklist
+labels: ["🚒 incident"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        A production incident started. Please stay cool, no need to call 911. Follow the checklist below - you've got this :muscle: <br/>
+        You may refer to the [Wing maintainers operational guide](https://www.winglang.io/docs/contributors/maintainers/operations) for more details.
+  - type: markdown
+    attributes:
+      value: |
+        ## Incident Response Checklist
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Checklist
+      description:
+      options:
+        - label: ACKNOWLEDGE the incident on PagerDuty and BetterStack.
+        - label: PRIORITIZE
+            P0 - Extreme/Total user interruption, no reasonable workaround.
+            P1 - Major degradation of user experience, with possible workarounds.
+            P2 - Minor degradation of user experience with workaround, or major contributor experience disruption.
```

Comment:
A nice enhancement could be to have PagerDuty incidents labeled automatically with a default priority, based on what triggered it. For example, when the canary fails, by default it's labeled as P0 since it usually means Wing can't be installed or a critical path isn't working. If the main build pipeline, it could be automatically labeled P1. Of course, in either case the oncall can downgrade the priority if they determine there's a reasonable workaround or it's a false alarm. (disclaimer: this was how oncall worked at Amazon)

Reply:
It's a nice enhancement, thank you!
For now I prefer to keep prioritization manual, though I believe we should automate as much as we can as our incident response process matures.